api/data_test.go

```diff
@@ -12,11 +12,13 @@ import (
 	"time"
 
 	"github.com/ONSdigital/dp-legacy-cache-api/api/mock"
+	errs "github.com/ONSdigital/dp-legacy-cache-api/apierrors"
 	"github.com/ONSdigital/dp-legacy-cache-api/models"
 	. "github.com/smartystreets/goconvey/convey"
 )
 
 var testCacheID = "testCacheID"
+var incorrectCacheID = "incorrectCacheID"
```

Comment:
This is only used in one test, so it should be moved where it's being used. The name is not right either, it should be something like `nonExistentCacheID`. The actual value should be a valid md5, otherwise this is going to need to be changed once the 400 ticket is implemented.